<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>934</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 233, 241);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="15" column="2">
        <widget class="QRadioButton" name="output1">
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Output Channel 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">output_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);</string>
         </property>
         <property name="text">
          <string>Mixer:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="3">
        <widget class="QRadioButton" name="output2">
         <property name="text">
          <string>Output Channel 2</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">output_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 11pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Choose Components:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="15">
        <widget class="QGraphicsView" name="output_mixer1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="out_region_radioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Outer Region</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">region_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="3" rowspan="15">
        <widget class="QGraphicsView" name="output_mixer2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="real_imaginary">
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Real/Imaginary</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">components_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 11pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Select Region:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QSlider" name="region_selecter">
         <property name="styleSheet">
          <string notr="true">/* 441F62
CE9BFA
D9B1FA
B6DDFE*/
QSlider::groove:horizontal {
    height: 8px;
    background: #B5A5F9; /* Updated lighter color */
    border-radius: 4px;
}
QSlider::handle:horizontal {
    background: #9385E5; /* Updated darker color */
    border: 2px solid #B5A5F9; /* Updated lighter color */
    width: 20px;
    height: 20px;
    margin: -6px 0;
    border-radius: 10px;
}
QSlider::sub-page:horizontal {
    background: #9385E5; /* Updated darker color */
    border-radius: 4px;
}
QSlider::add-page:horizontal {
    background: #B5A5F9; /* Updated lighter color */
    border-radius: 4px;
}
</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="in_region_radioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Inner Region</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">region_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="12" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="1">
        <widget class="QRadioButton" name="magnitude_phase">
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 9pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Magnitude/Phase</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">components_buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="imageFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="16" column="4">
        <widget class="QComboBox" name="Fourier_comboBox_3">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    color: rgb(255, 255, 255);
    background-color: #9688E9;  /* Background color */
    border: 1px solid #1C1C1C;
    border-radius: 15px;  /* Rounded corners */
    padding: 5px;
    font-size: 14px;
}
QComboBox::drop-down {
   
	color: #ffffff;
    border-radius: 10px;  /* Rounded corners for the arrow box */
    background-color: #9688E9;
    width: 30px;
}

QComboBox::down-arrow {
    width: 15px;
    height: 15px;
    color: white;  /* Arrow color */
}

QComboBox QAbstractItemView {
    border: 1px solid #1C1C1C;
    selection-background-color: #9688E9;  /* Selected item background color */
    background-color: #9688E9;  /* Dropdown background color */
    color: white;
    border-radius: 10px;
}</string>
         </property>
         <item>
          <property name="text">
           <string>FT Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="Glabel1">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 11pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Fourier Transform</string>
         </property>
        </widget>
       </item>
       <item row="13" column="4">
        <widget class="QGraphicsView" name="Gimage3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="13" column="6">
        <widget class="QGraphicsView" name="Gimage4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QGraphicsView" name="image1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <widget class="QGraphicsView" name="image4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QGraphicsView" name="Gimage2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QComboBox" name="Fourier_comboBox_2">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    color: rgb(255, 255, 255);
    background-color: #9688E9;  /* Background color */
    border: 1px solid #1C1C1C;
    border-radius: 15px;  /* Rounded corners */
    padding: 5px;
    font-size: 14px;
}
QComboBox::drop-down {
   
	color: #ffffff;
    border-radius: 10px;  /* Rounded corners for the arrow box */
    background-color: #9688E9;
    width: 30px;
}

QComboBox::down-arrow {
    width: 15px;
    height: 15px;
    color: white;  /* Arrow color */
}

QComboBox QAbstractItemView {
    border: 1px solid #1C1C1C;
    selection-background-color: #9688E9;  /* Selected item background color */
    background-color: #9688E9;  /* Dropdown background color */
    color: white;
    border-radius: 10px;
}</string>
         </property>
         <item>
          <property name="text">
           <string>FT Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(66, 31, 99);
font: 11pt &quot;Microsoft Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Input Table</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QGraphicsView" name="Gimage1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSlider" name="weight_1">
         <property name="styleSheet">
          <string notr="true">
QSlider::groove:vertical {
    width: 8px;
    background: #B5A5F9;
    border-radius: 4px;
}
QSlider::handle:vertical {
    background: #9385E5;
    border: 2px solid #B5A5F9;
    width: 20px;
    height: 20px;
    margin: 0 -6px;
    border-radius: 10px;
}
QSlider::sub-page:vertical {
    background: #9385E5;
    border-radius: 4px;
}
QSlider::add-page:vertical {
    background: #B5A5F9;
    border-radius: 4px;
}
</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QComboBox" name="Fourier_comboBox_1">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    color: rgb(255, 255, 255);
    background-color: #9688E9;  /* Background color */
    border: 1px solid #1C1C1C;
    border-radius: 15px;  /* Rounded corners */
    padding: 5px;
    font-size: 14px;
}
QComboBox::drop-down {
   
	color: #ffffff;
    border-radius: 10px;  /* Rounded corners for the arrow box */
    background-color: #9688E9;
    width: 30px;
}

QComboBox::down-arrow {
    width: 15px;
    height: 15px;
    color: white;  /* Arrow color */
}

QComboBox QAbstractItemView {
    border: 1px solid #1C1C1C;
    selection-background-color: #9688E9;  /* Selected item background color */
    background-color: #9688E9;  /* Dropdown background color */
    color: white;
    border-radius: 10px;
}</string>
         </property>
         <item>
          <property name="text">
           <string>FT Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="QGraphicsView" name="image3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="13" column="7">
        <widget class="QSlider" name="weight_4">
         <property name="styleSheet">
          <string notr="true">
QSlider::groove:vertical {
    width: 8px;
    background: #B5A5F9;
    border-radius: 4px;
}
QSlider::handle:vertical {
    background: #9385E5;
    border: 2px solid #B5A5F9;
    width: 20px;
    height: 20px;
    margin: 0 -6px;
    border-radius: 10px;
}
QSlider::sub-page:vertical {
    background: #9385E5;
    border-radius: 4px;
}
QSlider::add-page:vertical {
    background: #B5A5F9;
    border-radius: 4px;
}

</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QGraphicsView" name="image2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25px;
buttoon-shadow: 2px 2px 5px rgb(0, 0, 0);
widget-shadow: 2px 2px 5px rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="16" column="6">
        <widget class="QComboBox" name="Fourier_comboBox_4">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    color: rgb(255, 255, 255);
    background-color: #9688E9;  /* Background color */
    border: 1px solid #1C1C1C;
    border-radius: 15px;  /* Rounded corners */
    padding: 5px;
    font-size: 14px;
}
QComboBox::drop-down {
   
	color: #ffffff;
    border-radius: 10px;  /* Rounded corners for the arrow box */
    background-color: #9688E9;
    width: 30px;
}

QComboBox::down-arrow {
    width: 15px;
    height: 15px;
    color: white;  /* Arrow color */
}

QComboBox QAbstractItemView {
    border: 1px solid #1C1C1C;
    selection-background-color: #9688E9;  /* Selected item background color */
    background-color: #9688E9;  /* Dropdown background color */
    color: white;
    border-radius: 10px;
}</string>
         </property>
         <item>
          <property name="text">
           <string>FT Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FT Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QSlider" name="weight_2">
         <property name="styleSheet">
          <string notr="true">
QSlider::groove:vertical {
    width: 8px;
    background: #B5A5F9;
    border-radius: 4px;
}
QSlider::handle:vertical {
    background: #9385E5;
    border: 2px solid #B5A5F9;
    width: 20px;
    height: 20px;
    margin: 0 -6px;
    border-radius: 10px;
}
QSlider::sub-page:vertical {
    background: #9385E5;
    border-radius: 4px;
}
QSlider::add-page:vertical {
    background: #B5A5F9;
    border-radius: 4px;
}

</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="13" column="5">
        <widget class="QSlider" name="weight_3">
         <property name="styleSheet">
          <string notr="true">
QSlider::groove:vertical {
    width: 8px;
    background: #B5A5F9;
    border-radius: 4px;
}
QSlider::handle:vertical {
    background: #9385E5;
    border: 2px solid #B5A5F9;
    width: 20px;
    height: 20px;
    margin: 0 -6px;
    border-radius: 10px;
}
QSlider::sub-page:vertical {
    background: #9385E5;
    border-radius: 4px;
}
QSlider::add-page:vertical {
    background: #B5A5F9;
    border-radius: 4px;
}
</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(66, 31, 99);</string>
      </property>
      <property name="text">
       <string>FourierBlend Studio</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="region_buttonGroup"/>
  <buttongroup name="components_buttonGroup"/>
  <buttongroup name="output_buttonGroup"/>
 </buttongroups>
</ui>
